pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -264,6 +271,16 @@ let
     } // optionalAttrs pulseSupport {
       use_pulseaudio = true;
       link_pulseaudio = true;
+    } // optionalAttrs useOzone {
+      use_ozone = true;
+      ozone_platform_wayland = true;
+      ozone_platform_x11 = true;
+      ozone_auto_platforms = false;
+      use_xkbcommon = true;
+      use_glib = true;
+      use_system_libwayland = true;
+      use_system_minigbm = true;
+      use_system_libdrm = true;
```

Comment:
Does `system_wayland_scanner_path = "${wayland}/bin/wayland-scanner"` work (instead of using `substituteInPlace` above)?

Also based on the [documentation](https://source.chromium.org/chromium/chromium/src/+/80.0.3987.149:docs/ozone_overview.md?originalUrl=https:%2F%2Fcs.chromium.org%2F) we might also want `use_gtk=true`.